scrape/manager.go

```diff
@@ -147,16 +150,16 @@ func (m *Manager) reloader() {
 	defer ticker.Stop()
 
 	for {
+		select {
```

Comment:
I am not sure about this change.

Reply:
Can you elaborate what exactly is your concern?